src/structure_new/src/output/4C_structure_new_discretization_runtime_output_params.cpp

```diff
@@ -61,9 +62,11 @@ void Discret::Elements::StructureRuntimeOutputParams::init(
     auto io_strain =
         Teuchos::getIntegralValue<Inpar::Solid::StrainType>(io_parameter_list, "STRUCT_STRAIN");
     if (io_stress == Inpar::Solid::stress_none and io_strain == Inpar::Solid::strain_none)
+    {
       FOUR_C_THROW(
           "If stress / strain runtime output is required, one or two of the flags STRUCT_STRAIN / "
           "STRUCT_STRESS in the --IO section has to be set.");
+    }
```

Comment:
Technically, these flags are always set (but maybe not activated), since we work with default parameters for values, that the user did not specify in the input file:
```suggestion
    {
      FOUR_C_THROW(
          "If stress / strain runtime output is required, one or two of the flags STRUCT_STRAIN / "
          "STRUCT_STRESS in the --IO section has to be activated.");
    }
```